### PR TITLE
fix: updating migration repo for DSCrudAndBindings spec

### PR DESCRIPTION
**Pull request by brayn003**
## Description
Updates name of the migration repo in the following specs to the new name in TED
```
cypress/e2e/Regression/ClientSide/Git/ExistingApps/v1.9.24/DSCrudAndBindings_Spec.ts
```

## Automation
/ok-to-test tags="@tag.Git"

### :mag: Cypress test results

> [!TIP]
> 🟢 🟢 🟢 All cypress tests have passed! 🎉 🎉 🎉
> Workflow run: <https://github.com/appsmithorg/appsmith/actions/runs/11100011900>
> Commit: 7eedfb48b9182ffe226eea2da00c5af55d32b861
> <a href="https://internal.appsmith.com/app/cypress-dashboard/rundetails-65890b3c81d7400d08fa9ee5?branch=master&workflowId=11100011900&attempt=1" target="_blank">Cypress dashboard</a>.
> Tags: `@tag.Git`
> Spec:
> <hr>Mon, 30 Sep 2024 06:03:18 UTC

## Communication
Should the DevRel and Marketing teams inform users about this change?
- [ ] Yes
- [ ] No

## Summary by CodeRabbit

- **Bug Fixes**
	- Updated repository names in test specifications to ensure accurate references during application validation and Git Autocommit tests.
	- Added a new migration tag to categorize tests under the migration context.
	- Modified the list of limited tests to include new test files for Git-related functionalities.

### CODE REVIEW

**Comment by sagar-qa007** [comment]
@brayn003 Please run the test here.

**Comment by brayn003** [comment]
@sagar-qa007 The test is dependant on a custom TED build. Running the tests here with ok-to-test will not yield favourable result

**Comment by brayn003** [comment]
Dependent on https://github.com/appsmithorg/TestEventDriver/pull/79

File: app/client/cypress/e2e/Regression/ClientSide/Git/ExistingApps/v1.9.24/DSCrudAndBindings_Spec.ts [hunk 1]
```diff
@@ -261,51 +216,6 @@ describe(
       agHelper.WaitUntilToastDisappear("Delete customer successful!");
       agHelper.ClickButton("Close");
       agHelper.AssertElementAbsence(locators._modal);
-      agHelper.Sleep(2000);
-    });
-
-    it.skip("4. Edit JSObject & Check Updated Data ", () => {
```
Comment (sagar-qa007) [inline app/client/cypress/e2e/Regression/ClientSide/Git/ExistingApps/v1.9.24/DSCrudAndBindings_Spec.ts:267]: Do we not need this test case? 

@brayn003

File: app/client/cypress/e2e/Regression/ClientSide/Git/ExistingApps/v1.9.24/DSCrudAndBindings_Spec.ts [hunk 2]
```diff
@@ -176,10 +121,9 @@ describe(
       table.ReadTableRowColumnData(0, 4).then(($cellData) => {
         expect($cellData).to.eq("Active");
       });
-      agHelper.Sleep(500);
     });
 
-    it("3. Validate widgets & bindings", () => {
+    it("Validate widgets & bindings", () => {
```
Comment (sagar-qa007) [inline app/client/cypress/e2e/Regression/ClientSide/Git/ExistingApps/v1.9.24/DSCrudAndBindings_Spec.ts:126]: Please add the sequence so report will be easy to check. Kindly do for each it.

**Review by sagar-qa007 — CHANGES_REQUESTED** [review]

File: app/client/cypress/e2e/Regression/ClientSide/Git/ExistingApps/v1.9.24/DSCrudAndBindings_Spec.ts (hunk 1 above)
Comment (brayn003) [reply app/client/cypress/e2e/Regression/ClientSide/Git/ExistingApps/v1.9.24/DSCrudAndBindings_Spec.ts:267]: This was skipped for a long time. Hence, removed it

**Review by sagar-qa007 — APPROVED** [review]

**Pull request merged**